### PR TITLE
Update obsrv core

### PR DESCRIPTION
**Pull request by SadanandGowda**

### CODE REVIEW

File: terraform/modules/helm/flink/flink-helm-chart/values.yaml [hunk 1]
```diff
@@ -480,11 +480,46 @@ unified-pipeline:
 
   job_classname: in.sanketika.obsrv.pipeline.task.UnifiedPipelineStreamTask
 
+transformer-ext:
```
Comment (anandp504) [inline terraform/modules/helm/flink/flink-helm-chart/values.yaml:483]: @SadanandGowda Why is transformer-ext configuration added in obsrv-automation?

File: terraform/modules/helm/velero/variables.tf [hunk 2]
```diff
@@ -104,6 +104,7 @@ variable "velero_backup_bucket_region" {
 variable "velero_sa_iam_role_name" {
   type        = string
   description = "Service Account IAM Role Name"
+  default     = ""
```
Comment (anandp504) [inline terraform/modules/helm/velero/variables.tf:107]: Why do we need an emptry String as default?

File: terraform/modules/helm/velero/variables.tf [hunk 3]
```diff
@@ -121,4 +122,5 @@ variable "velero_sa_annotations" {
 variable "gcp_project_id" {
   type        = string
   description = "Google Project ID, if the provider is gcp"
-}
\ No newline at end of file
+  default     = ""
```
Comment (anandp504) [inline terraform/modules/helm/velero/variables.tf:125]: Why do we need an emptry String as default?

File: terraform/gcp/main.tf [hunk 4]
```diff
@@ -355,9 +355,10 @@ module "flink" {
   env                            = var.env
   building_block                 = var.building_block
   flink_container_registry       = var.flink_container_registry
-  flink_release_name             = var.flink_release_name
-  flink_image_name               = var.flink_image_name
   flink_image_tag                = var.flink_image_tag
+  flink_merged_pipeline_release_names = var.flink_merged_pipeline_release_names
```
Comment (anandp504) [inline terraform/gcp/main.tf:359]: The list of variables need to be formatted properly.

**Review by anandp504 — CHANGES_REQUESTED** [review]
Please address the PR comments

File: terraform/modules/helm/flink/flink-helm-chart/values.yaml (hunk 1 above)
Comment (SadanandGowda) [reply terraform/modules/helm/flink/flink-helm-chart/values.yaml]: Moved this transformer-ext and unified-pipeline ext to flink.tfpl

File: terraform/modules/helm/velero/variables.tf (hunk 2 above)
Comment (SadanandGowda) [reply terraform/modules/helm/velero/variables.tf:107]: needed regarding gcp changes

File: terraform/modules/helm/velero/variables.tf (hunk 3 above)
Comment (SadanandGowda) [reply terraform/modules/helm/velero/variables.tf:125]: needed regarding gcp changes, same as velero_sa_iam_role_name

File: terraform/gcp/main.tf (hunk 4 above)
Comment (SadanandGowda) [reply terraform/gcp/main.tf:359]: Formatted accordingly

**Review by anandp504 — APPROVED** [review]
Approved 👍

**Pull request merged**